documentation/src/main/asciidoc/topics/ref_redis_commands.adoc

```diff
@@ -67,6 +67,9 @@ The {brandname} RESP endpoint implements the following Redis commands:
 | link:https://redis.io/commands/mset[MSET]
 |
 
+| link:https://redis.io/commands/linsert[LINSERT]
+| Current implementation time complexity is O(N), where N is the size of the list.
```

Comment:
Do we mention that it uses the first match with the pivot when traversing from left to right? Maybe not necessary, but I am bringing this up because Redis does not make the behavior explicit in the command doc.